smsdk/client_v0.py

```diff
@@ -1039,7 +1037,9 @@ def clean_query_downtime_titles(self, query):
             else:
                 func = ''
                 if '__' in key:
-                    key, func = key.split('__')
+                    splitted_key = key.split('__')
```

Comment:
Earlier splitting was not working for downtime as for `machine__source__in` key split will return 3 values and we were handling only 2, so changed it as per others ( like cycle)